packages/math-input/src/components/keypad-button.tsx

```diff
@@ -154,10 +149,12 @@ class KeypadButton extends React.PureComponent {
         }
 
         const borderStyle = [];
-        if (borders.indexOf(BorderDirections.LEFT) !== -1) {
+        if (borders.includes(BorderDirections.LEFT)) {
+            // @ts-expect-error TS2345
             borderStyle.push(styles.leftBorder);
```

Comment:
I couldn't figure out how to make TS happy here. It seemed to think this code was unreachable and I didn't want to spend for ever on this trying to verify.